packages/docgen/src/fragments/FunctionSignatureTypeFragment.ts

```diff
@@ -54,15 +55,19 @@ export const FunctionSignatureTypeFragment = (
       : ''
   }) => ${getLabelForType(type, { wrap: false })}`
 
-  if (wrap) {
-    return `
+  if (wrap === 'none') {
+    return content
+  }
+
+  if (wrap === 'code-block') {
+    return codifyValue(content)
+  }
+
+  return `
```

Comment:
Why are there so many changes to `docgen/src/*`? It does not seem related to `core`.

Reply:
I don't know, but as long as the reference documentation works that's fine by me.
You can ask Szilard if it bugs you

Reply:
ok

Reply:
I think you know what this was for @plmercereau, because you raised the issue to me, for which the fix was created. 😄

Things were breaking in DocGen after moving things from `core` to `hasura-auth-js` and `hasura-storage-js`. Some type references were unavailable with the previous setup, but they started working when we moved things. These type references were not properly covered by the auto-generator, so I had change a few things under the hood.

Reply:
right, I did not follow this was related. Everything works, so that's fine